### DISCUSSION TITLE
Custom Theming (Please help)

### DISCUSSION THREAD

**Discussion by harinathpamu** [tailwindlabs/tailwindcss#3002] [Help] [answered]
This works fine but ....
```HTML
<!-- Dark mode enabled -->
<html class="dark">
<body>
  <!-- Will be black -->
  <div class="bg-white dark:bg-black">
    <!-- ... -->
  </div>
</body>
</html>
```
But I need something like this below
```HTML
<!--Blue Theme mode enabled -->
<html class="bluetheme">
<body>
  <!-- Will be blue -->
  <div class="bg-white bluetheme:bg-blue">
    <!-- ... -->
  </div>
</body>
</html>
```

Please help me on how to solve ....

**Comment by simonswiss** [comment, marked as answer]
Hey 👋

The current implementation of `darkMode` is not meant to work as a multi-theme approach.

Here's a very basic demo of how you could handle this: https://github.com/adamwathan/theming-tailwind-demo

More context on the "dark mode only" nature of our `darkMode` implementation, copied from the [dark mode PR](https://github.com/tailwindlabs/tailwindcss/pull/2279):

## Design Rationale

The obvious question when implementing this feature is _"why not build some generalized universal theming solution that supports an arbitrary number of themes?"_

The main reasons are:

1. Operating systems only support light and dark.
2. Supporting more than a single theme introduces significantly more complexity and challenging decisions.
3. 99% of people only care about dark mode.
4. We can always add a universal theming feature down the road that supersedes this if we like.

So for now, this feature is unapologetically **dark mode only**. Every design decision has been made with the mindset of _"we only support light mode and dark mode and we don't care about supporting anything else"_.

That's why the parent class is simply `dark`, the prefix is just `dark`, the configuration option is just `dark`, etc.

**Reply by harinathpamu** [reply]
Thanks for your reply : )
